src/Consumer.php

```diff
@@ -61,21 +54,67 @@ public function __construct()
      * @access public
      * @return void
      */
-    public function start(callable $consumer = null, $isBlock = true)
```

Comment:
@nmred I've removed `$isBlock = true` because I didn't see any reason to start consumers without initialising the event loop. We can discuss this, obviously